src/routes/user/user.routes.js

```diff
@@ -22,6 +26,11 @@ const userRoutes = (router) => {
 
     return res.status(StatusCodes.CREATED).json(response);
   });
+
+  router.get('/logout', auth, (req, res) => {
+    res.clearCookie('auth');
+    return res.redirect('/api/');
```

Comment:
This should probably return a successful response (204 or 200 if response has a body) as logout in the form of cookie deletion should be carried out on the front-end.